google/cloud/managedidentities/v1beta1/resource.proto

```diff
@@ -64,7 +64,7 @@ message Domain {
   map<string, string> labels = 2;
 
   // Optional. The full names of the Google Compute Engine
-  // [networks](/compute/docs/networks-and-firewalls#networks) the domain
```

Comment:
// [networks](https://cloud.google.com/compute/docs/networks-and-firewalls#networks) the domain